### PR TITLE
Add an example for throwing a trident

### PR DESCRIPTION
**Pull request by firejoust**
A trident is a unique weapon in itself, so I added an example on how it can be used with mineflayer

### CODE REVIEW

**Comment by Camezza** [comment]
(P.S. I want a free t-shirt from digitalocean so please add a "hacktoberfest-accepted" label to this PR)

**Comment by rom1504** [comment]
can you move it to a examples/pvp folder ?

**Comment by Camezza** [comment]
fixed

File: examples/pvp/trident.js [hunk 1]
```diff
@@ -0,0 +1,37 @@
+const mineflayer = require('mineflayer')
+const projectile = require('mineflayer-projectile')
+const bot = mineflayer.createBot() // etc. (https://github.com/PrismarineJS/mineflayer/blob/master/docs/api.md#mineflayercreatebotoptions)
```
Comment (rom1504) [inline examples/pvp/trident.js:3]: can you add options similar to other examples ?

**Pull request closed without merging**